tests/BaseTest.php

```diff
@@ -76,6 +77,9 @@ protected function _before()
 				'wp_verify_nonce' => function () {
 					return true;
 				},
+				'is_email' => function (string $email) {
+					return $email === 'invalid' ? false : true;
```

Comment:
```suggestion
					return $email === 'invalid';
```

Reply:
this is actually not ok, because the logic is reversed

Reply:
Then use `!==` instead 😄

Reply:
Fixed